OSDOCS-15874: adds physically bound image building to bootc docs Micr… (#97670)
openshift/openshift-docs: microshift_install_bootc/microshift-install-bootc-physically-bound.adoc

:_mod-docs-content-type: ASSEMBLY
[id="microshift-install-bootc-physically-bound"]
include::_attributes/attributes-microshift.adoc[]
= Creating a fully self-contained bootc image
:context: microshift-install-bootc-physically-bound

toc::[]

> Some configurations require operating system images that are tightly integrated with the hardware. If your edge-computing use case involves embedded systems on specialized devices, high security, or high hardware control scenarios, using a physically bound bootc image might be ideal.

Author: Do we have anything to say for physically-bound bootc images that set them apart from a basic RHEL for Edge image? We need to provide a use case.

> Some configurations require operating system images that are tightly integrated with the hardware. If your edge-computing use case involves embedded systems on specialized devices, high security, or high hardware control scenarios, using a physically bound bootc image might be ideal.

Reviewer: RHEL for Edge image is usually an ISO image. 
The PBI images are container images.

:_mod-docs-content-type: ASSEMBLY
[id="microshift-install-bootc-physically-bound"]
include::_attributes/attributes-microshift.adoc[]
= Creating a fully self-contained bootc image
:context: microshift-install-bootc-physically-bound

toc::[]

> If your use case requires that your bootc image includes everything needed to run workloads, a physically-bound container image might be ideal. Edge-computing scenarios involving embedded systems on specialized devices, high security, or high hardware control scenarios are likely candidates.

Reviewer: ```suggestion
If your use case requires that your bootc image includes everything needed to run workloads, a physically-bound container images are what you need to implement. Edge-computing scenarios involving embedded systems on specialized devices, high security, or high hardware control scenarios are likely candidates.
```
